src/main/java/aquarium/shop/order/OrderController.java

```diff
@@ -0,0 +1,29 @@
+package aquarium.shop.order;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class OrderController {
+
+    @Autowired
+    private OrderService orderService;
+
+    @PostMapping("/order")
+    public void addOrder(OrderDetails orderDetails) {
+        orderService.addOrder(orderDetails);
+    }
+
+    @PatchMapping("/order/payment")
+    public void changeOrderPaymentStatus(@RequestBody OrderDetails orderDetails ) {
```

Comment:
Ok, let's leave it like that, but it not the best solution to pass whole object just to get id. I would prefer something like:
@RequestBody Integer id
but not sure right now if this will work. I will check that later.